tools/clang/lib/SPIRV/EmitSPIRVAction.cpp

```diff
@@ -1108,6 +1108,28 @@ class SPIRVEmitter : public ASTConsumer {
     }
   }
 
+  /// Processes the 'asfloat', 'asint', and 'asuint' intrinsic functions.
+  uint32_t processIntrinsicAsType(const CallExpr *callExpr, spv::Op spvOp) {
```

Comment:
I'd suggest to rename `spvOp` as `toType` or something similar. It would make the signature more clear.